hooks/index.php

```diff
@@ -4,4 +4,5 @@
 require_once __DIR__ . '/accordion.php';
 require_once __DIR__ . '/search.php';
 require_once __DIR__ . '/pages.php';
-require_once __DIR__ . '/wpsync.php';
\ No newline at end of file
+require_once __DIR__ . '/wpsync.php';
+require_once __DIR__ . '/woocommerce.php';
```

Comment:
I think my only thought is that it probably wouldn't be a bad idea to check if woocommerce is active before requiring.

```php
add_action('plugins_loaded', function () {
  if (class_exists('\WooCommerce')) {
    require_once __DIR__ . '/woocommerce.php';
  }
});
```